### ISSUE TITLE
Enabling IK resets pose in other armature

### ISSUE THREAD

**Issue by joepal1976** [makehumancommunity/makehuman-plugin-for-blender#58]
To demonstrate:

* Import human A
* Pose human A
* Import human B
* Click add IK on human B

Expected outcome: human A remains in current pose and is unaffected

Actual outcome: all bones in human A armature are reset back to rest pose

**Comment by Palmer-JC** [comment]
I get the same thing.  The first 3 lines of op are
```python
bpy.ops.object.mode_set(mode='POSE')
bpy.ops.pose.select_all(action='SELECT')
bpy.ops.pose.transforms_clear()
```
I put a `return` after line 2, and the select all does both skeletons:
![both selected](https://user-images.githubusercontent.com/7916131/63524086-5aa8e700-c4c9-11e9-8bb8-a3fc8d0c5a52.jpg)

I wonder if this is supposed to happen, or an issue with 2.80?  I happen to have a 2,79b for win32 still around, which  I used to check the loading / running of DLL's on 32 bit.  This does not happen on an old version, but one where you still are bringing characters across via socket.  FYI, current version does not even load on 2.79 anymore.  It gets `invalid syntax` when class properties are declared using ':'.

The `select_all` is required to get the B-BONE scaling to work.  It looks kind of bad if IK is applied when posed,  Clear all afterward worked.  It does error on removal of IK though.  Am going to look into removal

**Comment by Fakey-Person** [comment]
No, it happens maybe, because it can not do that pose with Via Ik/FK enabled. 
because trying to pose something trying to use an arm or forearm (it's not possible) because you only need to move the hand to move the arm http://www.makehumancommunity.org/forum/posting.php?mode=reply&f=22&t=19086&sid=6dc701a2f8e72dab2b6cbb0f68040bc3#
